qa/rspec/matchers/be_installed.rb

```diff
@@ -14,3 +13,9 @@
     subject.removed?(subject.hosts, subject.name)
   end
 end
+
+RSpec::Matchers.define :be_removed do
```

Comment:
Why having this matcher twice? not sure I understand why this is being highlight here.